### PR TITLE
[Hedley] Update to v15

### PR DESCRIPTION
**Pull request by pratikpc**
**Describe the pull request**

- #### What does your PR fix?  
Updates Hedley to 15

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
Yes

### CODE REVIEW

File: ports/hedley/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "hedley",
+  "version-string": "15",
+  "description": "A C/C++ header to help move #ifdefs out of your code"
```
Comment (JackBoosY) [inline ports/hedley/vcpkg.json:4]:
```suggestion
  "description": "A C/C++ header to help move #ifdefs out of your code",
  "homepage": "https://nemequ.github.io/hedley/"
```

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/hedley/vcpkg.json (hunk 1 above)
Comment (pratikpc) [reply ports/hedley/vcpkg.json]: @JackBoosY awesome!

**Review by JackBoosY — APPROVED** [review]

File: ports/hedley/vcpkg.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "hedley",
+  "version-string": "15",
```
Comment (strega-nil-ms) [inline ports/hedley/vcpkg.json:3]:
```suggestion
  "version": "15",
```

Comment (pratikpc) [reply ports/hedley/vcpkg.json]: @strega-nil-ms you might also have to run vcpkg x-add-one or else it will break

**Comment by JackBoosY** [comment]
Thanks for this PR!

**Pull request merged**